README.md

```diff
@@ -111,10 +111,11 @@ make check
   - Type `k9s` and press enter
   - Press `0` to select all namespaces
   - Wait for all pods to be in the `Running` state (look for the `STATUS` column)
-  - Use the arrow key to select `nsga-memory` then press the `l` key to view logs from the pod
+  - Use the arrow key to select `imdb` pod then press the `l` key to view logs from the pod
+  - Press `1` to go to the beginning of logs
   - To go back, press the `esc` key
   - Use the arrow key to select `jumpbox` then press `s` key to open a shell in the container
-    - Hit the `ngsa-memory` NodePort from within the cluster by executing `http ngsa-memory:8080/version`
+    - Hit the `IMDB-App` NodePort from within the cluster by executing `http ngsa-memory:8080/version`
     - Verify 200 status in the response
```

Comment:
This http command needs to be updated (i.e. not ngsa-memory) and it currently fails I'm assuming because jumpbox is running in the default namespace.

Reply:
(same issue in the jumpbox section below)

Reply:
Updated the http command for now, we will have to expose the service to default namespace to make work with jumpbox. Will look in to that.

Reply:
you can just use the dns resolution - http://pod.namespace.svc.cluster.local:8080

Reply:
updated and resolved